extensions/amp-story-auto-analytics/0.1/auto-analytics-configs.js

```diff
@@ -21,11 +21,6 @@
 export const buildGtagConfig = (gtagId) => ({
   'vars': {
     'gtag_id': gtagId,
-    'config': {
-      [gtagId]: {
-        'groups': 'default',
-      },
-    },
```

Comment:
Have you checked with the people maintaining the internal dashboards that this doesn't break them?

Reply:
Verified, thank you ❤️